internal/setup/setup.go

```diff
@@ -659,6 +704,23 @@ func parsePackage(baseDir, pkgName, pkgPath string, data []byte) (*Package, erro
 	return &pkg, err
 }
 
+func GetGeneratePath(path string) (dir string, err error) {
```

Comment:
This should just be `validateGeneratePath` or similar, and it should be private as its purpose is both trivial and unclear... and untested!

Reply:
Agree, this was a byproduct of splitting the PR because this function was used from another module but that is no longer the case.

Done in https://github.com/canonical/chisel/pull/143/commits/b8f91bfb79bb77fea0c42f3e83f7aed56c33785e.